panel/src/components/Forms/Field/DateField.vue

```diff
@@ -80,6 +81,16 @@ export default {
     focus() {
       this.$refs.input.focus();
     },
+    onBlur(event) {
+      // close the calendar dropdown on blur on the input
+      // but make sure the click target wasn't the calendar itself
+      if (
+        this.$refs.calendar &&
+        this.$el.contains(event.relatedTarget) === false
```

Comment:
```suggestion
        this.$el.contains(event.target) === false
```
Can you try this Like it works for me.